owasp-dependency-check-suppressions.xml

```diff
@@ -479,4 +479,13 @@
     <packageUrl regex="true">^pkg:maven/org\.asynchttpclient/async-http-client-netty-utils@2.5.3$</packageUrl>
     <cve>CVE-2021-43138</cve>
   </suppress>
+
+  <suppress>
+    <!-- Suppress cves that aren't applicable to hadoop client -->
```

Comment:
It looks like we already have a section for suppressions like this in https://github.com/apache/druid/blob/691e26d2429cd81afef092dbda32e24ad56bb510/owasp-dependency-check-suppressions.xml#L322-L329

Can you combine this with that section.

This comment explaining why the CVE is being suppressed is great!